client/components/Assignments/AssignmentGroupList.tsx

```diff
@@ -248,9 +254,9 @@ class AssignmentGroupListComponent extends React.Component {
                             <div className="sd-list-header__number sd-flex-grow">
                                 <span className="a11y-only">{gettext(
                                     'Number of Assignments: ',
-                                    {count: totalCount}
+                                    {count: (filteredAssignments?.length ?? 0)}
```

Comment:
would be good to update this to use placeholder based translation. It should be possible to pass a react component like `gettext('Number of Assignments: {{n}}', {n: () => <div>5</div>})`

Reply:
We can create a ticker for this if you want to update it

Reply:
it's 2 min maybe you can update without a ticket? :)